tfhe/benches/integer/bench.rs

```diff
@@ -1002,10 +1251,14 @@ criterion_group!(
     scalar_mul_parallelized,
     scalar_div_parallelized,
     scalar_rem_parallelized,
+    scalar_div_rem_parallelized,
```

Comment:
probably only div_rem

Reply:
In the case of scalar division, div is done on its own, rem needs division

In ciphertext, ciphertext  division, they are both computed by div_rem, so whether you need only the div or remainder, the cost is the same

Reply:
So what should we do for scalar ? div and div_rem ?

Reply:
probably the 3 ops for scalar, div is its own thing, rem uses div, and div_rem does both

Reply:
For scalars `(div_rem` == rem) != div`
For ciphertexts `div == rem == div_rem`

Reply:
I think we are almost good now with this last push.